src/lib/components/Select.react.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+
+import React, {Component} from 'react';
+import { default as MuiSelect } from '@material-ui/core/Select';
+import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+type Props = {
+    id: string,
+    autoFocus?: boolean,
+    autoWidth: boolean,
+    classes?: {},
+    disabled?: boolean,
+    inputRef?: () => void,
+    MenuProps?: Object,
+    multiple: boolean,
+    options?: Array<Object>,
+    /** Dash callback to update props on the server */
+    setProps?: () => void,
+    name?: string,
+    native: boolean,
+    onChange?: () => void,
+    onClose?: () => void,
+    onOpen?: () => void,
+    open?: boolean,
+    readOnly?: boolean,
+    renderValue?: Node,
+    tabIndex?: number,
+    value?: T,
+    variant?: 'standard' | 'outlined' | 'filled',
+};
+
+type State = {
+    dataSourceRender: Array<string>,
+    selectedValue: string,
+};
+
+const defaultProps = {
+    autoWidth: false,
+    classes: {},
+    disabled: false,
+    multiple: false,
+    options: [],
+    setProps: () => {},
+    value: null,
+    variant: 'standard',
+};
+
+export default class Select extends Component<Props, State> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            selectedValue: this.props.selectedValue,
+            dataSourceRender: this.props.dataSource,
```

Comment:
Make sure every prop is defined, and any props that are not required are given default values.